[fix] add directions on link grouping (#7688)
EddieHubCommunity/LinkFree: pages/docs/how-to-guides/links.mdx

| icon  | string | true     | The icon you would like displayed. We support the following icon sets: **Fa** and **Si** from React Icons.                                                                                     |
| group | string | false    | Optional property to group links together, any without a group will be added to the end in a new group called "Others". If no groups are used then no groups are displayed including "Others". |

> To add more links, add another object inside the links collection. For example:

Reviewer: I have added the line break so `For example` is on a new line, to match the project's documentation
```suggestion
To add more links add another object inside the links collection. 

For example:
```

_Clicks on these social shortcuts that match any of your links will also increment their respective counters_

> ### Grouping your Links 

Reviewer 2: @CBID2 What do you think about reiterating here either in the title or description that this is an optional feature.  Some folks may just scroll quickly and miss that this is optional farther up

Author: Ooh good idea!

> To place your social links into categories, you can use names like `primary`,  `secondary`, `business`, or `personal`in the `group` string. Here's an example:

Reviewer 3: The names do not have to be like codenames, I think it would be good to add a example over here like the ones mentioned in the example

```suggestion
To place your social links into categories, you can use names like `Business`,  `Socials Primary`, or `:sparkles: Personal` in the `group` string.

These can include any alphanumeric, unicode characters and symbols.
```

+++I think we should also attach a screenshot to tell them how the text they enter inside the `group` field will be displayed

Author: Good point @krishstwt! :)

### Grouping your Links (optional)
To place your social links into categories, you can use names like `Business`,  `Socials Primary`, or `:sparkles: Personal` in the `group` string.

These can include any alphanumeric, unicode characters and symbols.

<ClipboardCopy>
```js
"links": [
 {
      "group": "Business",
      "name": "Business: DevRel Services",
      "url": "http://eddiejaoude.io",
      "icon": "FaLink"
    },
    {
      "group": "Socials Primary",
      "name": "Twitter: Follow me",
      "url": "https://twitter.com/eddiejaoude",
      "icon": "FaTwitter",
      "color": "#00ACEE"
    },
    {
      "group": "Socials Primary",
      "name": "YouTube: Learn more about Open Source",
      "url": "https://youtube.com/eddiejaoude",
      "icon": "FaYoutube",
      "color": "#FF0000"
    },
    {
      "group": "Socials Primary",
      "name": "GitHub: Collaborate on Open Source",
      "url": "https://github.com/eddiejaoude",
      "icon": "FaGithub"
    },
    {
      "group": "Socials Primary",
      "name": "LinkedIn: Let's connect",
      "url": "https://linkedin.com/in/eddiejaoude",
      "icon": "FaLinkedin"
    },

]
   
  ```

> </ClipboardCopy>

Reviewer: As Markdown (and MDX) is sensitive to line breaks this needs the following:

```suggestion
</ClipboardCopy>

```

<ClipboardCopy>
```js
"links": [
 {
      "group": "Business",
      "name": "Business: DevRel Services",
      "url": "http://eddiejaoude.io",
      "icon": "FaLink"
    },
    {
      "group": "Socials Primary",
      "name": "Twitter: Follow me",
      "url": "https://twitter.com/eddiejaoude",
      "icon": "FaTwitter",
      "color": "#00ACEE"
    },
    {
      "group": "Socials Primary",
      "name": "YouTube: Learn more about Open Source",
      "url": "https://youtube.com/eddiejaoude",
      "icon": "FaYoutube",
      "color": "#FF0000"
    },
    {
      "group": "Socials Primary",
      "name": "GitHub: Collaborate on Open Source",
      "url": "https://github.com/eddiejaoude",
      "icon": "FaGithub"
    },
    {
      "group": "Socials Primary",
      "name": "LinkedIn: Let's connect",
      "url": "https://linkedin.com/in/eddiejaoude",
      "icon": "FaLinkedin"
    },

]
   
  ```
</ClipboardCopy>

Here's how the categories look on our website:
![Group-categories](https://github.com/EddieHubCommunity/LinkFree/assets/105683440/813f1706-b4f9-4ff3-be2b-0a7a4954737b)
Here's how they look with the complete profile: 

> ![LinkFree profile with grouped links](https://user-images.githubusercontent.com/624760/214538171-099cac71-d2f7-49a8-b3bb-2747506a1bac.png)

Reviewer: I've added a line break here between the Markdown and Javascript which will fix the build failure:

```suggestion
![LinkFree profile with grouped links](https://user-images.githubusercontent.com/624760/214538171-099cac71-d2f7-49a8-b3bb-2747506a1bac.png)

```

### Grouping your Links (optional)
To place your social links into categories, you can use names like `Business`,  `Socials Primary`, or `:sparkles: Personal` in the `group` string.

These can include any alphanumeric, unicode characters and symbols.

<ClipboardCopy>
```js
"links": [
 {
      "group": "Business",
      "name": "Business: DevRel Services",
      "url": "http://eddiejaoude.io",
      "icon": "FaLink"
    },
    {
      "group": "Socials Primary",
      "name": "Twitter: Follow me",
      "url": "https://twitter.com/eddiejaoude",
      "icon": "FaTwitter",
      "color": "#00ACEE"
    },
    {
      "group": "Socials Primary",
      "name": "YouTube: Learn more about Open Source",
      "url": "https://youtube.com/eddiejaoude",
      "icon": "FaYoutube",
      "color": "#FF0000"
    },
    {
      "group": "Socials Primary",
      "name": "GitHub: Collaborate on Open Source",
      "url": "https://github.com/eddiejaoude",
      "icon": "FaGithub"
    },
    {
      "group": "Socials Primary",
      "name": "LinkedIn: Let's connect",
      "url": "https://linkedin.com/in/eddiejaoude",
      "icon": "FaLinkedin"
    }

]
   
  ```
</ClipboardCopy>

> Here's how they look with the complete profile: 

Reviewer: Improving the wording:

```suggestion
Here is how the links with grouping will appear on your Profile:
```

### Grouping your Links (optional)
To place your social links into categories, you can use names like `Business`,  `Socials Primary`, or `:sparkles: Personal` in the `group` string.

These can include any alphanumeric, unicode characters and symbols.

<ClipboardCopy>
```js
"links": [
 {
      "group": "Business",
      "name": "Business: DevRel Services",
      "url": "http://eddiejaoude.io",
      "icon": "FaLink"
    },
    {
      "group": "Socials Primary",
      "name": "Twitter: Follow me",
      "url": "https://twitter.com/eddiejaoude",
      "icon": "FaTwitter",
      "color": "#00ACEE"
    },
    {
      "group": "Socials Primary",
      "name": "YouTube: Learn more about Open Source",
      "url": "https://youtube.com/eddiejaoude",
      "icon": "FaYoutube",
      "color": "#FF0000"
    },
    {
      "group": "Socials Primary",
      "name": "GitHub: Collaborate on Open Source",
      "url": "https://github.com/eddiejaoude",
      "icon": "FaGithub"
    },
    {
      "group": "Socials Primary",
      "name": "LinkedIn: Let's connect",
      "url": "https://linkedin.com/in/eddiejaoude",
      "icon": "FaLinkedin"

>     },

Reviewer: This comma is invalid JSON

```suggestion
    }
```
